docs/configure-artifact-repository.md

```diff
@@ -75,12 +75,15 @@ artifacts:
 
 ## Configuring AWS S3
 
-Create your bucket and access keys for the bucket. AWS access keys have the same
-permissions as the user they are associated with. In particular, you cannot
-create access keys with reduced scope. If you want to limit the permissions for
-an access key, you will need to create a user with just the permissions you want
-to associate with the access key. Otherwise, you can just create an access key
-using your existing user account.
+First, create your bucket:
+
+```bash
+export mybucket=bucket249
+aws s3 mb s3://$mybucket [--region xxx]
+```
+
+Next, create a policy which will be saved locally.
+You will attach this to the relevant authentication method in one of the sections below.
```

Comment:
So this isn't used by all of them anymore, right? Since S3 Access Grants don't need this. We may want to subdivide IAM and non-IAM access? i.e. have a heading "AWS S3 IAM Access" or something and have the IAM methods all underneath it

Reply:
```suggestion
You will attach this in one of the sections below based on your chosen authentication method.
```

reword and simplify a bit

Reply:
Yep, sounds good, I'll do that.